### DISCUSSION TITLE
Ability to choose audio source in Control Center

### DISCUSSION THREAD

**Discussion by semi710** [ErikReider/SwayNotificationCenter#282] [Q&A] [answered]
Anyone able to create a config that can control all possible output device channels, like KDE or gnome have for their control center?

**Comment by ErikReider** [comment]
Could you explain further? I'm not really sure of what you mean 😅

**Reply by semi710** [reply]
As gnome control center or KDE Control center have volume control in multiple layers 
e.g

<details>
    <summary>
        VOL
    </summary>
    <details>
        <summary>Volume: --------------</summary>
        Chrome: --------------<br>
        Firefox: --------------
    </details>
    <details>
        <summary>
            Volume Channel:
        </summary>
            [ ] Speaker <br>
            [ ] Headphone <br>
            [ ] Some other source 
    </details>
</details>

**Reply by ErikReider** [reply]
Do you mean something like the volume widget? Just realized that the man page doesn't include it so that should be fixed now :)

**Reply by semi710** [reply]
Does that widget have multiple volume control or only the current source?

**Reply by ErikReider** [reply, marked as answer]
> Does that widget have multiple volume control or only the current source?

Not currently, only the default source. It looks like this:
![image](https://github.com/ErikReider/SwayNotificationCenter/assets/35975961/85cc0a8f-00ec-4f66-a568-3ca32bebc7a8)

**Reply by semi710** [reply]
This looks nice.
is it planned to have other sources to be listed there?

**Reply by ErikReider** [reply]
Maybe in the future
